var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -514,6 +525,9 @@ def setup_dependent_package(self, module, dependent_spec):
     def setup_build_environment(self, env):
         spec = self.spec
         if "+cuda" in spec and "+wrapper" in spec:
+            # Add more debug options for CUDA
+            if spec.variants["build_type"].value in ["RelWithDebInfo", "Debug"]:
+                env.append_flags("CXXFLAGS", "-Xcompiler -rdynamic -lineinfo")
```

Comment:
Under what circumstances is this needed? Does it have particular CUDA compatibility constraints? Shouldn't it be CUDA flags, not CXX flags? And did you know that you can set custom CXX flags through spack? I would shy away from flags that aren't required to get the build to pass.

Reply:
This basically just adds stuff that acts like `-g` but for CUDA when doing builds with debug symbols. I'm not familiar enough with how I would do this with CUDA flags since `nvcc_wrapper` handles this for us. I will just remove this for now. It's not strictly necessary, just convenient.

Reply:
Ah man, it's wrappers on top of wrappers on top of wrappers!